ui/js/controllers/perf/compare.js

```diff
@@ -58,6 +59,66 @@ perf.controller('CompareChooserCtrl', [
                 $scope.newRevision = tip;
             };
 
+            $scope.getPreviousRevision = function() {
+                $scope.proposalRevision = $scope.newRevisionError = null;
+
+                // only check for a full revision
+                if ($scope.newRevision.length != 12) return;
+
+                // indicate that we are loading if it takes some time
+                var timeout = setTimeout(function() {
+                    $scope.proposalRevisionLoading = true;
+                }, 400);
+
+                var promise;
+                if ($scope.newProject.name == "try") {
+                    // try require some special logic
+                    var iProjs = _.filter($scope.projects, function(proj) {
+                        return proj.name == "mozilla-inbound" ||
+                            proj.name == "mozilla-central" ||
+                            proj.name == "fx-team";
```

Comment:
@wlach Oh yeah, this is quite what we wanted (or maybe better actually). In there we have
```json
["mozilla-inbound", "mozilla-central", "fx-team", "b2g-inbound"]
```

So basically, b2g-inbound is added in comparison to my patch. @jmaher is that ok if we add that branch ?

Also, it is used here:
```
ui/js/controllers/perf/graphs.js:809:                                            testsDisplayed, options, thPerformanceBranches) {
ui/js/controllers/perf/graphs.js:907:        thPerformanceBranches.forEach(function (branch) {
```

The meaning of this value should be the same everywhere (obviously), but I'm not qualified to judge that. Is is good ? Else I could create a new *value* for this specific need.